operational/aws/tag_cardinality/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.1
+
+- policy now reports cardinality proper instead of tag values directly
```

Comment:
Did you mean to say 'properly' rather than 'proper'? Same for the other changelog file

Reply:
No. See definition 4 here: https://www.collinsdictionary.com/us/dictionary/english/proper